stations.csv

```diff
@@ -21505,7 +21505,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 23518;Möllbrücke-Sachsbg;mollbrucke-sachsbg;8103862;;;;;AT;Europe/Vienna;f;f;f;f;f;f;;;f;;f;8100419;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv23518;
 23519;Mühldorf-Möllbrücke;muhldorf-mollbrucke;8102126;;;;;AT;Europe/Vienna;f;f;f;f;f;f;;;f;;f;8100087;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv23519;
 23520;Mürzzuschlag;murzzuschlag;8103144;;47.607611;15.677128;;AT;Europe/Vienna;f;f;f;t;f;f;;;f;;f;8100029;t;;f;;f;;f;;f;;f;8100029;t;;f;;f;;;;;f;;f;;f;;f;ATACA;t;;f;f;;;;;;;;;;;;;;;;Мюрццушлаг;;;米爾茨楚施拉格;urn:trainline:public:nloc:csv23520;
-23521;Nenzing;nenzing;8101233;;;;;AT;Europe/Vienna;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;Ненцинг;;;嫩青;urn:trainline:public:nloc:csv23521;
+23521;Nenzing;nenzing;8101233;;;;;AT;Europe/Vienna;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;49;t;f;;;;;;;;;;;;;;;;Ненцинг;;;嫩青;urn:trainline:public:nloc:csv23521;
```

Comment:
This station is missing some important infos like coordinates, and is not suggestable.
I think this should be changed.